CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact sandstone module, newest changes on the top.
 
+## [unreleased]
+
+### Added
+
+- `sandstone/VideoPlayer` support for playback speed adjustment
```

Comment:
You should mention the newly added api `setPlaybackSpeed`. Please refer to the previous changelog.

Reply:
I updated to include about new api.